tracks/r/exercises/word-count/mentoring.md

```diff
@@ -0,0 +1,31 @@
+### Reasonable solution
+
+This solution is simple and follows a logical thought process.
+
+```r
+word_count <- function(input) {
+  input_no_punctuation <- gsub('[[:punct:]]+','',input)
+  input_parsed_whitespace <- gsub('\\s+', ' ', trimws(input_no_punctuation))
```

Comment:
Since you used `…no_…` above, would it make the variable content clearer here as well? `…parsed_…` seems very generic to be (not a native English-speaker, though).

Reply:
Good suggestion!